packages/truffle-core/lib/commands/test.js

```diff
@@ -66,8 +86,14 @@ const command = {
       Environment.detect(config).catch(done);
     }
 
-    let ipcDisconnect;
-    let files = [];
+    // enables in-test debug() interrupt, forcing compileAll
+    if (config.debug) {
+      config.compileAll = true;
+    }
+
+    var ipcDisconnect;
+
+    var files = [];
```

Comment:
```suggestion
    let files = [];
```

Reply:
so you're the one throwing `var`'s everywhere :trollface:

Reply:
oh jeez I hope not. bad merge perhaps?